src/encoded/static/store.js

```diff
@@ -50,7 +50,7 @@ export const reducers = {
         return state;
     },
 
-    'browseBaseState' : function(state='only_4dn', action){
+    'browseBaseState' : function(state='all', action){
```

Comment:
Changes default state of '[Show] External Data' toggle to be true.

Reply:
But on UI side only, also default redirection from /browse/ (and other things like, metadata.tsv, barplot aggs (I think..)) changed to exclude `&award.project=4DN` param in search.py.

Reply:
If this PR will not be merged shortly, default External Data = true feature could be merged separately into master.